### DISCUSSION TITLE
Error banner is telling me to contact pages team?

### DISCUSSION THREAD

**Discussion by kion-dgl** [community/community#60715] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Trying to deploy a GitHub page site on my subdomain `mml2.dashgl.org`. A records are set to the right IP address. Not sure if I need to add a TXT record or something to allow for HTTPS.

- A record 185.199.108.153
- AAAA record 2606:50c0:8000::153

I get an error banner prompting me to get in touch with the Pages team.

![Screenshot 2023-07-13 at 04-20-47 Build software better together](https://github.com/community/community/assets/25621780/7849f909-355b-4c9b-942b-06cc6017b3a8)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @kion-dgl ! The best way to contact the Pages is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
